pages/_document.js

```diff
@@ -0,0 +1,16 @@
+import Document, { Html, Main, NextScript } from "next/document";
+
+class ShurikenDocument extends Document {
```

Comment:
The deployment preview is broken, and I think it has something to do with this special component. Probably if you really need this, you need to import here global css files.